app/uk/gov/hmrc/cgtpropertydisposalsfrontend/connectors/EmailVerificationConnector.scala

```diff
@@ -59,7 +61,11 @@ class EmailVerificationConnectorImpl @Inject() (
     val protocol = getEmailVerificationConfig[String]("protocol")
     val host     = getEmailVerificationConfig[String]("host")
     val port     = getEmailVerificationConfig[String]("port")
-    s"$protocol://$host:$port/email-verification/verification-requests"
+    val domain   = getEmailVerificationConfig[String]("domain")
+    if (domain =!= "")
```

Comment:
very hacky-ish - but not sure of any other clean way to achieve it

Reply:
The idea is that the domain key's value will be blank in all environments except development and staging.

Reply:
to avoid the if-else in the code you could change the config values to be either `"/"` or `"/cgt-property-disposals-stubs"` - then you can just do :
```scala
 s"$protocol://$host:$port$domain/email-verification/verification-requests"
```
which might not look 100% right, but it saves you having to do this check in the code

Reply:
one other option could be to make the upload submit to a test-only endpoint in our FE - that way the stub won't need to move to the public zone and we don't need to make these config changes. The stub would need to know the full url of that FE endpoint though to return in the response to the initiate call, but that's just reading something from config

Reply:
it will also mean running the service with test-only routes when running locally and when the QAs run the service in Jenkins

Reply:
I think the test-only routes are already enabled in some of the environments - the acceptance tests require it already.